README.md

```diff
@@ -32,3 +32,26 @@ Mac/Linux: $HOME/.vscode/extensions
 ```
 For example, to setup a symlink on Mac/Linux: `ln -s /path/to/rls_vscode/ ~/.vscode/extensions/rls_vscode`
 Restart VSCode in order to load the extension. More information available via [VSCode docs](https://code.visualstudio.com/Docs/extensions/example-hello-world#_installing-your-extension-locally).
+
+## Troubleshooting
+
+### something .. something ... `tsc -watch -p ./` or `ENOSPC`
```

Comment:
well... the text around that is magical and incomprehensible to the unskilled javascript mind (like mine)